lib/active_record/connection_adapters/cockroachdb/database_tasks.rb

```diff
@@ -0,0 +1,19 @@
+require "active_record/base"
+
+module ActiveRecord
+  module ConnectionAdapters
+    module CockroachDB
+      class DatabaseTasks < ActiveRecord::Tasks::PostgreSQLDatabaseTasks
+        def structure_dump(filename, extra_flags=nil)
```

Comment:
since you don't care about these arguments, you could write `structure_dump(*)` instead. same applies below.